src/people.rs

```diff
@@ -214,28 +276,31 @@ impl Index {
             lookup: None,
             indexer: Box::new(move |context: &Context, person_id: PersonId| {
                 let value = context.get_person_property(person_id, property);
-                IndexValue::compute(&value)
+                (IndexValue::compute(&value), format!("{value:?}"))
```

Comment:
Architecture: I'm not sure we want to compute the string every time we do an index. This is in an inner loop. I would prefer instead to have a separate formatter and then only compute the string the first time.

Are you sure we want to use Debug and not Display here?